packages/react-i18n/src/utilities/translate.tsx

```diff
@@ -118,6 +120,10 @@ function translateWithDictionary(
     if (typeof count === 'number') {
       const group = new Intl.PluralRules(locale).select(count);
       result = result[group];
+
+      additionalReplacements[PLURALIZATION_KEY_NAME] = new Intl.NumberFormat(
```

Comment:
Why not doing `replacements.count = new Intl.NumberFormat(locale).format(count);` here?

Reply:
I started by doing that, but I realized it was modifying the original replacement dictionary passed by the caller.

Reply:
Oh true... 👍